sdk/translation/azure-ai-translation-text/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 1.0.0b2 (Unreleased)
+  - Synch and generate from API spec repo, update tests and samples.
```

Comment:
My understanding is that this should capture the changes for users. Can you confirm that messages like this are expected?

Reply:
If we want to use this message, please change "Synch" to "Synchronize" or "Sync".

Reply:
I only added that change because you asked me to in the last version of the pull request.  I will remove it.

Reply:
Reverted file and pushed.